### PR TITLE
Feat: Support spread operator in logs

### PR DESCRIPTION
**Pull request by Meldiron**
Allows multiple params in log and write methods.

Runtimes:

- [x] Bun
- [x] CPP (Doesnt seem supported by language)
- [x] Dart (Doesnt seem supported by language)
- [x] Deno
- [x] Dotnet (Not supported by language)
- [x] Go
- [x] Java
- [x] Kotlin
- [x] Node
- [x] PHP
- [x] Python
- [x] Ruby
- [x] Swift (Not supported by language)

### CODE REVIEW

File: runtimes/go/versions/latest/src/go.mod [hunk 1]
```diff
@@ -4,5 +4,5 @@ go 1.22.5
 
 replace openruntimes/handler v0.0.0 => /usr/local/build
 
-require openruntimes/handler v0.0.0
-require github.com/open-runtimes/types-for-go/v4 v4.0.5
+// require openruntimes/handler v0.0.0
```
Comment (Meldiron) [inline runtimes/go/versions/latest/src/go.mod:7]: Remove?!? Seems unneeded?!? This is so good, it was always hussle with this when devleoping

File: runtimes/go/versions/latest/src/go.mod [hunk 2]
```diff
@@ -4,5 +4,5 @@ go 1.22.5
...
+require github.com/open-runtimes/types-for-go/v4 v4.0.6-0.20240818164233-b4d5d5df2b30
```
Comment (Meldiron) [inline runtimes/go/versions/latest/src/go.mod:8]: Release version

File: tests/resources/functions/go/latest/go.mod [hunk 3]
```diff
@@ -4,7 +4,7 @@ go 1.22.5
 
 require (
 	github.com/go-resty/resty/v2 v2.11.0
-	github.com/open-runtimes/types-for-go/v4 v4.0.5
+	github.com/open-runtimes/types-for-go/v4 v4.0.6-0.20240818164233-b4d5d5df2b30
```
Comment (Meldiron) [inline tests/resources/functions/go/latest/go.mod:7]: Release version

**Review by christyjacob4 — APPROVED** [review]

**Comment by abnegate** [comment]
@Meldiron This is supported in the some of the other languages:

C++: `void log(const std::string &message, ...)`
.NET: `public void Log(params object[] message)`
Swift: `func log(_ message: Any...)`

Might be because of a terminology mix-up: these are called variadic parameters, spread operator is what allows you to expand an array into it's individual items. Similar, but used in different contexts.

**Pull request merged**